src/styles/_colors.scss

```diff
@@ -1,11 +1,13 @@
 html {
-  --base-color-background: #282c34;
-  --base-color-link: #61dafb;
-  --base-color: white;
+	--base-color-background: #282c34;
+	--base-color-link: #61dafb;
+	--base-color: white;
+	--base-color-button: #929292;
```

Comment:
I think these custom properties must be declared in `:root` to be available in all document.
And if you can please define all colors variables from the design and these variables can be used in these lines.
I attach the list of all colors in the design:
![095BE094-EFE7-4E00-B344-FD168D747E1E](https://user-images.githubusercontent.com/3743350/89130438-68812d80-d4ca-11ea-9472-1786778293a7.png)

let me know if not clear and if you have any doubt.

Reply:
it's a good suggestion,  but 1 question at do we have the negative/contrast of these colors I mean if @andresV74  put
`--background-card: white,` on the for the dark mode on the tag `html[data-theme="dark"] ` should be --background-card: black, if the color didn't change keep it

Reply:
I already did the changes and suggestions that you made and push those changes to gitHub, but I'm not quite sure if I have to make a new PR with those changes. Please let me know